epictrack-web/src/components/shared/index.tsx

```diff
@@ -425,3 +453,28 @@ export const ETPreviewBox = ({ children, sx, ...rest }: HeaderProps) => {
     </Box>
   );
 };
+
+interface DashedBorder {
+  children?: React.ReactNode | string;
+  [prop: string]: unknown;
+  sx?: SxProps;
+}
+export const DashedBorder = ({ children, sx, ...rest }: DashedBorder) => {

```

Comment:
I've actually added the preview text and previewbox for this? Could you check them?